priv/test_repo/migrations/20201028042738_migrate_resources2.exs

```diff
@@ -0,0 +1,20 @@
+defmodule AshPostgres.TestRepo.Migrations.MigrateResources2 do
+  @moduledoc """
+  Updates resources based on their most recent snapshots.
+
+  This file was autogenerated with `mix ash_postgres.generate_migrations`
+  """
+
+  use Ecto.Migration
+
+  def up() do
+    create table(:multitenant_orgs, primary_key: false) do
+      add(:id, :binary_id, null: true, default: fragment("uuid_generate_v4()"), primary_key: true)
```

Comment:
`null: false` ?
I noticed that IDs allow `nil` in many places, is that intended?

Reply:
Really its just up to the resource wether or not its possible to be null, but it wouldn't really make sense in most contexts